phy/utils/_misc.py

```diff
@@ -98,3 +100,17 @@ def _show_shortcuts(shortcuts, name=''):
     for name in sorted(shortcuts):
         print('{0:<40}: {1:s}'.format(name, _show_shortcut(shortcuts[name])))
     print()
+
+def _git_version():
```

Comment:
well git is installed on travis (or we can easily install it), so you can just call this function and check that it returns a string with the expected number of characters...